src/GCLogger.cpp

```diff
@@ -0,0 +1,38 @@
+#include "memory.h"
+
+
+GCLogger::GCLogger(const char* fileName):
+	m_logFile(fileName, std::fstream::out)
+{}
+
+GCLogger::~GCLogger(){
+    m_logFile.close();
+}
+
+
+void GCLogger::writeLogLine(TMemoryManagerEvent event){
+    m_logFile << event.begin.toString(SNONE, 3)
+              << ": [" << event.eventName << " ";
+    if(!event.heapInfo.empty()){
+        TMemoryManagerHeapInfo eh = event.heapInfo;
+        m_logFile << eh.usedHeapSizeBeforeCollect << "K->"
```

Comment:
You should `/ BYTES_IN_KB` right here. Use `enum` right inside this member function, forget about defines.